__tests__/groups/group.test.js

```diff
@@ -245,7 +245,7 @@ describe('Discord Groups Page', () => {
   });
 
   test('Should display only specified groups when name=<group-name> with different case', async () => {
-    const groupNames = 'fIrSt,DSA+COdInG';
+    const groupNames = 'fIrSt,COdInG';
```

Comment:
- is this change required?

Reply:
I have added the fix to search for the prefix of all words in the same discord group name, So I have modified the test to support that also.

- The discord group name is "DSA-Coding-Group", previously I was searching for "DSA coding" which is the prefix of the group name and it will show up.
- Now this current PR fix, will also support showing up if I search for "coding" only